fred2/shipgoalsdlg.cpp

```diff
@@ -1008,6 +1031,7 @@ void ShipGoalsDlg::update_item(int item, int multi)
 		case AI_GOAL_STAY_NEAR_SHIP:
 		case AI_GOAL_STAY_STILL:
 		case AI_GOAL_CHASE_SHIP_CLASS:
+		case AI_GOAL_CHASE_SHIP_TYPE:
```

Comment:
Farther down this file, around line 1159, there is a `case TYPE_SHIP_CLASS:` that needs a corresponding `case TYPE_SHIP_TYPE:`